src/oca_pre_commit_hooks/checks_odoo_module_xml.py

```diff
@@ -271,7 +272,8 @@ def check_xml_dangerous_qweb_replace_low_priority(self):
                         self.checks_errors["xml-dangerous-qweb-replace-low-priority"].append(
                             f'{manifest_data["filename_short"]}:{child.sourceline} '
                             'Dangerous use of "replace" from view '
-                            f"with priority `{priority} < {DFTL_MIN_PRIORITY}`"
+                            f"with priority `{priority} < {DFTL_MIN_PRIORITY}`. "
+                            'Only replace as a last resort. Consider making the block invisible instead (t-if="0")'
```

Comment:
- `t-if="False"`
- All tips in the above message are also applicable: modify attributes and position="move" also work here. Maybe the message could be the same

Reply:
Thanks for the help, I have made the changes